build.proj

```diff
@@ -33,8 +33,10 @@
     <ExeExtension>.exe</ExeExtension>
     <ExeExtension Condition=" '$(OS)' != 'Windows_NT' "></ExeExtension>
     <DotnetStage0>$(Stage0Path)/dotnet$(ExeExtension)</DotnetStage0>
-
     <DotnetCliBuildDirectory>$(RepoRoot)/build_projects/dotnet-cli-build</DotnetCliBuildDirectory>
+    <OutputDirectory>$(RepoRoot)/artifacts</OutputDirectory>
+    <PackagesDir>$(RepoRoot)/.nuget</PackagesDir>
```

Comment:
Sure, we could definitely do that...I'm just concerned that it would make the variables a little more magic since they would be tucked away in yet another file. I'm not opposed to putting them into a different file if you think that would be more valuable